src/aiida_quantumespresso/workflows/pw/relax.py

```diff
@@ -31,13 +23,16 @@ def define(cls, spec):
         """Define the process specification."""
         # yapf: disable
         super().define(spec)
-        spec.expose_inputs(PwBaseWorkChain, namespace='base',
+        spec.expose_inputs(PwBaseWorkChain, namespace='base_relax',
             exclude=('clean_workdir', 'pw.structure', 'pw.parent_folder'),
             namespace_options={'help': 'Inputs for the `PwBaseWorkChain` for the main relax loop.'})
-        spec.expose_inputs(PwBaseWorkChain, namespace='base_final_scf',
+        spec.expose_inputs(PwBaseWorkChain, namespace='base_init_relax',
```

Comment:
Nitpick: Maybe move this before the `base_relax` namespace since chronologically it will be used before.